yb-voyager/src/queryissue/queryissue.go

```diff
@@ -95,6 +95,9 @@ func (p *ParserIssueDetector) GetDMLIssues(query string) ([]issue.IssueInstance,
 	if err != nil {
 		return nil, fmt.Errorf("error parsing query: %w", err)
 	}
+	if queryparser.IsViewObject(parseTree) || queryparser.IsMviewObject(parseTree) {
+		return nil, nil
+	}
```

Comment:
@priyanshi-yb, can we have some higher level check to find out if its a `ddl` or `dml` as the name of the function also, rather than doing it based on the object type in here.